common/djangoapps/third_party_auth/tests/test_decorators.py

```diff
@@ -20,11 +21,7 @@ def mock_view(_request):
     return HttpResponse()
 
 
-# remove this decorator once third_party_auth is enabled in CMS
```

Comment:
I had asked about this before and there is no intention of ever enabling tpa in CMS at this point, so this comment is just unnecessary. Also we have so many decorators for the same reason everywhere there is no point in having this comment lol.